hedera-node/src/test/java/com/hedera/services/queries/crypto/GetAccountBalanceAnswerTest.java

```diff
@@ -87,10 +96,15 @@
 
 	@BeforeEach
 	private void setup() {
```

Comment:
I dont think that you need all those changes. Only the `scheduleStore` mock in order to add it in the constructor when you initialise `StateView`